### PR TITLE
Updated links to angular-formly-website, added DuckieTV description + logo and DuckieDocs logo

### PR DESCRIPTION
**Pull request by SchizoDuckie**
I also found another bug, but couln't get to the bottom of that (es6 templating somethingsomething?) see attached screenshot.

image

### CODE REVIEW

File: app/states/root/children/users/components/data/users-data/new/description.md~ [hunk 1]
```diff
@@ -0,0 +1,10 @@
+You can add your project to this list by following these steps:
```
Comment (kentcdodds) [inline app/states/root/children/users/components/data/users-data/new/description.md~]: Guessing you didn't mean to include this file...

File: app/states/root/children/users/components/data/users.js [hunk 2]
```diff
@@ -29,11 +29,18 @@ export default [
   {
     name: 'DuckieDocs',
     slug: 'duckiedocs',
-    logo: require('./users-data/no-logo.png'),
+    logo: require('./users-data/duckiedocs/logo.png'),
```
Comment (kentcdodds) [inline app/states/root/children/users/components/data/users.js:32]: woot woot :-)

**Comment by kentcdodds** [comment]
Once you get rid of that extra file, I'll definitely merge this, rebuild the site, and deploy! :-)

File: app/states/root/children/users/components/data/users-data/new/description.md~ (hunk 1 above)
Comment (SchizoDuckie) [inline app/states/root/children/users/components/data/users-data/new/description.md~]: uh where did that come from. cursed be you nano!

I'll fix and resubmit!

**Comment by SchizoDuckie** [comment]
Fixed! All hail `git commit --amend` and `git push -f`

**Comment by kentcdodds** [comment]
Like it never happened :-)

**Comment by kentcdodds** [comment]
I'll need to rebuild the site later when I get to the computer. I'll let you know when it's up.

**Comment by SchizoDuckie** [comment]
cool stuff :+1: take your time!

**Pull request merged**